src/studio/src/designer/frontend/language/transform-old-format-to-new.js

```diff
@@ -0,0 +1,19 @@
+const fs = require('fs');
```

Comment:
Is this needed in the future?
Can't this might as well just be deleted, assuming the script has already served its purpose?